apps/super-app/src/common/components/navbar/NavBar.css

```diff
@@ -0,0 +1,44 @@
+@import url("https://fonts.googleapis.com/css?family=Hind");
+a:visited {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px 32px 10px;
```

Comment:
I suggest using `rem` as the unit instead of `px`. `rem` scales a bit better do different screen sizes.

Reply:
Also, do we need the bottom padding? The mockup looks like the icons are vertically centered.